liquibase-core/src/main/java/liquibase/statement/ExecutablePreparedStatementBase.java

```diff
@@ -201,7 +200,34 @@ private void applyColumnParameter(PreparedStatement stmt, int i, ColumnConfig co
         } else {
             // NULL values might intentionally be set into a change, we must also add them to the prepared statement
             LOG.debug(LogType.LOG, "value is explicit null");
-            stmt.setNull(i, java.sql.Types.NULL);
+            if (col.getType() == null) {
+                stmt.setNull(i, java.sql.Types.NULL);
+                return;
+            }
+            if (col.getType().toLowerCase().contains("date") || col.getType().toLowerCase().contains("time")) {
```

Comment:
Do we need this check anymore? Or is it also take care of by the alias logic below?